daisy_workflows/image_import/debian/translate.py

```diff
@@ -67,13 +67,15 @@ def DistroSpecific(g):
       logging.warn('Could not uninstall Azure agent. Continuing anyway.')
 
     utils.update_apt(g)
-    pkgs = ['google-cloud-packages-archive-keyring', 'google-cloud-sdk',
-            'google-compute-engine']
-    # Debian 8 doesn't support the new guest agent, so we need to install
-    # the legacy Python version.
```

Comment:
Any consequences for the old Guest agent and running on python 2.7/3.4?

Reply:
AFAIK there aren't big changes planned for the old guest agent.